tls/s2n_handshake_io.c

```diff
@@ -635,6 +636,42 @@ static int handshake_read_io(struct s2n_connection *conn)
     return 0;
 }
 
+static void s2n_write_keylog(struct s2n_connection *conn)
+{
+    char *keylog_fn;
+    FILE *keylog_fp;
+
+    keylog_fn = getenv("SSLKEYLOGFILE");
+    if (!keylog_fn) {
+        return;
+    }
+
+    keylog_fp = fopen(keylog_fn, "a");
```

Comment:
I think that this should be fopen(... "ab"); ... for older systems that need to be told that this is binary data.